_includes/header.html

```diff
@@ -1,80 +1,60 @@
-                            </li>
-                            <li class="nav-item ms-lg-auto">
-                                <a class="nav-link nav-link-icon me-2" href="https://github.com/RSEToolkit/training"
-                                    target="_blank">
-                                    <i class="fa fa-github me-1"></i>
-                                    <p class="d-inline text-sm z-index-1 font-weight-bold" data-bs-toggle="tooltip"
-                                        data-bs-placement="bottom" title="Star us on Github">Github</p>
-                                </a>
-                            </li>
-                            <li class="nav-item my-auto ms-3 ms-lg-0">
-
-                                <a href="#" class="btn btn-sm  bg-gradient-primary  mb-0 me-1 mt-2 mt-md-0">Get
-                                    Started</a>
-
-                            </li>
-                        </ul>
-                    </div>
-                </div>
-            </nav>
-            <!-- End Navbar -->
-        </div>
-    </div>
-</div>
-
-<header class="header-2">
-    <div class="page-header min-vh-75 relative"
-        style="background-image: url('./assets/img/ryan-stone-U3cctUBucn0-unsplash.jpg')">
-        <span class="mask bg-gradient-primary opacity-4"></span>
+{% if page.layout == "home" %}
+<nav class="navbar navbar-expand-lg position-absolute top-0 z-index-3 w-100 shadow-none my-3  navbar-transparent ">
+    {% else %}
+    <nav class="navbar navbar-expand-lg navbar-light bg-white py-3">
+        {% endif %}
         <div class="container">
-            <div class="row">
-                <div class="col-lg-7 text-center mx-auto">
-                    <h1 class="text-white pt-3 mt-n5">RSE Competency Toolkit</h1>
-                    <p class="lead text-white mt-3">Shared, collaborative and progressive learning paths for roles in
-                        the RSE
-                        community.</p>
+            <a class="navbar-brand text-white" href="{% link index.md %}" rel="tooltip" title="RSE Competencies Toolkit"
+                data-placement="bottom">
+                <img src="../assets/img/1529053464.svg" alt="down-arrow" class="arrow ms-2 d-lg-block d-none">
+            </a>
+            <button class="navbar-toggler shadow-none ms-2" type="button" data-bs-toggle="collapse"
+                data-bs-target="#navigation" aria-controls="navigation" aria-expanded="false"
+                aria-label="Toggle navigation">
+                <span class="navbar-toggler-icon mt-2">
+                    <span class="navbar-toggler-bar bar1"></span>
+                    <span class="navbar-toggler-bar bar2"></span>
+                    <span class="navbar-toggler-bar bar3"></span>
+                </span>
+            </button>
+            <div class="collapse navbar-collapse w-100 pt-3 pb-2 py-lg-0 ms-lg-12 ps-lg-5" id="navigation">
+                <ul class="navbar-nav navbar-nav-hover ms-auto">
```

Comment:
Recommend adding a 'Home' link to the homepage to this nav, otherwise you can't get back to the homepage on a narrow screen as the logo link disappears